### PR TITLE
[keras/layers/preprocessing/normalization.py] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

**Review by gbaned — APPROVED** [review]

File: keras/layers/preprocessing/normalization.py [hunk 1]
```diff
@@ -52,11 +52,12 @@ class Normalization(base_preprocessing_layer.PreprocessingLayer):
           example, if shape is `(None, 5)` and `axis=1`, the layer will track 5
           separate mean and variance values for the last axis. If `axis` is set
           to `None`, the layer will normalize all elements in the input by a
-          scalar mean and variance. Defaults to -1, where the last axis of the
+          scalar mean and variance. When `-1` the last axis of the
```
Comment (qlzh727) [inline keras/layers/preprocessing/normalization.py:55]: I think we should add a ',' after the "When `-1`".

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/layers/preprocessing/normalization.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/layers/preprocessing/normalization.py]: Done

**Review by qlzh727 — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @SamuelMarks Can you please resolve conflicts? Thank you!

**Comment by SamuelMarks** [comment]
Fixed

**Pull request closed without merging**